doc/12-icinga2-api.md

```diff
@@ -1208,13 +1213,13 @@ In addition to these parameters a [filter](12-icinga2-api.md#icinga2-api-filters
 The following example acknowledges all services which are in a hard critical state and sends out
 a notification for them:
 
-```
-$ curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
+```bash
+curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
  -X POST 'https://localhost:5665/v1/actions/acknowledge-problem' \
  -d '{ "type": "Service", "filter": "service.state==2&service.state_type=1", "author": "icingaadmin", "comment": "Global outage. Working on it.", "notify": true, "pretty": true }'
 ```
 
-```
+```json
```

Comment:
Invalid JSON yet again. Can you please check yourself that the diff renders properly in the files changed tab, so that we can avoid further round trips should this be the case in even more code blocks.